lib/message.rb

```diff
@@ -39,6 +39,8 @@ class HL7::Message
   attr :segment_delim
   attr :delimiter
 
+  POSSIBLE_ENCODINGS = ["UTF-8", "ISO-8859-1"]
```

Comment:
Just for documentation reasons how about naming this `ALLOWED_ENCODINGS` or even `HL7_STANDARD_ENCODINGS`

Reply:
I think this is related to the comment below. Virtually any encoding could be expected despite the fact that these 2 are commonly used as stated [here](https://wiki.hl7.org/index.php?title=Character_Set_used_in_v2_messages).

I've only chosen two that are much more likely (or conceivable). Hence the name. Using `ALLOWED_ENCODINGS` on these two means nothing else is allowed. This is true if we have an exhaustive list.

Reply:
But isn't your link an exhaustive list of all the encodings existing in our markets?